extra/nrnivmodl_core_makefile.in

```diff
@@ -186,11 +188,11 @@ $(MOD_OBJS_DIR)/%.obj: $(MOD_TO_CPP_DIR)/%.ispc | $(MOD_OBJS_DIR)
 
 # translate MOD files to ISPC using NMODL
 $(mod_ispc_files): $(MOD_TO_CPP_DIR)/%.ispc: $(MODS_PATH)/%.mod | $(MOD_TO_CPP_DIR)
-	$(NMODL_ENV_VAR) $(NMODL_BINARY_PATH) $< -o $(MOD_TO_CPP_DIR)/ @nmodl_arguments_ispc@
+	$(NMODL_ENV_VAR) $(NMODL_BINARY_PATH) $< -o $(MOD_TO_CPP_DIR)/ @nmodl_arguments_ispc@ $(NMODL_FLAGS_RUNTIME)
 
 # translate MOD files to CPP using mod2c/NMODL
 $(mod_cpp_files): $(MOD_TO_CPP_DIR)/%.cpp: $(MODS_PATH)/%.mod | $(MOD_TO_CPP_DIR)
-	$(NMODL_ENV_VAR) $(NMODL_BINARY_PATH) $< -o $(MOD_TO_CPP_DIR)/ @nmodl_arguments_c@
+	$(NMODL_ENV_VAR) $(NMODL_BINARY_PATH) $< -o $(MOD_TO_CPP_DIR)/ @nmodl_arguments_c@ $(NMODL_FLAGS_RUNTIME)
```

Comment:
I am wondering what should happen when `nmodl_arguments_c` is `codegen --force passes --verbatim-rename --inline sympy --analytic` for example and then we want to add `sympy --pade passes --unroll` or something else. Should the `$(NMODL_FLAGS_RUNTIME)` maybe overwrite the original flags?

Reply:
to summarize the private conversation:

we agreed that if we want a flag always active (and forget about it), we put it in the build. Otherwise they can go in the runtime flags

Reply:
We should also take care of the order of the flags though. Providing to `nmodl` the following flags won't work:
```
./nmodl <mod_file> codegen --force passes --verbatim-rename --inline sympy --analytic sympy --pade passes --unroll
```
It needs to be:
```
./nmodl <mod_file> codegen --force passes --verbatim-rename --inline --unroll sympy --analytic --pade
```

Reply:
discussed offline - we will overwrite all flags